pkg/scannerv4/mappers/mappers.go

```diff
@@ -826,10 +779,6 @@ func nvdSeverityAndScores(vuln *claircore.Vulnerability, v *nvdschema.CVEAPIJSON
 func vulnerabilityName(vuln *claircore.Vulnerability) string {
 	// Attempt per-updater patterns.
 	switch {
-	case rhelUpdaterPattern.MatchString(vuln.Updater):
-		if v, ok := findName(vuln, rhelVulnNamePattern); ok {
-			return v
-		}
```

Comment:
Confirming: With VEX, the name is indeed the vuln name, not title, descriptions, etc...

Reply:
yeah, VEX data is CVE-centric, so the names will be CVE-20XX-XXXX